mbta.rb

```diff
@@ -1,13 +1,40 @@
 # Your solution here
 class Subway
   def stops_between_stations(start_line, start_station, end_line, end_station)
+  attr_reader :Subway
+  @subway = subway
```

Comment:
Usually, the attr are added in the class not methods within the class. Also always make sure that attr has the same name as your instance variable